portal/code/cx-portal-shared-components/src/components/basic/CustomIcons/index.tsx

```diff
@@ -0,0 +1,20 @@
+import CustomIcons from './Icons.svg';
```

Comment:
I'm thinking if we really should add custom icons at all. For the upload we could use MUI Cloud Upload icon, although it looks a bit different.

https://mui.com/components/material-icons/?query=uplo&theme=Outlined&selected=CloudUploadOutlined

The reason is that I think it's hard to maintain custom SVGs without probably breaking something in the future.

Reply:
@oyo @jjeroch is it okay for you? we can also discussed it at the meeting

Reply:
@christianraysono dona as two static icons components